examples/12-GP-filter-example.ipynb

```diff
@@ -4,12 +4,19 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    noise = TimeSeries.from_values(np.random.normal(0, NOISE_DISTANCE, SAMPLE_SIZE))
</pre><p>You could potentially consider using darts.utils.timeseries_generation, which allows to genereate sine series as well as Gaussian series.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/385/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>